### PR TITLE
Healing Sting

### PR DESCRIPTION
**Pull request by Ahbahl**
Adds a healing sting for changelings
Adds an icon for healing sting

http://www.llagaming.net/forums/threads/healing-sting.3931/ <- Feedback thread
http://www.llagaming.net/forums/threads/changeling-healing-sting.3697/ <- Suggestion thread

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 1]
```diff
@@ -178,4 +178,25 @@ obj/effect/proc_holder/changeling/sting/cryo
...
+obj/effect/proc_holder/changeling/sting/heal/sting_action(var/mob/user, var/mob/living/carbon/target)
+	add_logs(user, target, "stung", object="healing sting")
+	target << "You feel a tiny prick."
+	spawn(0)
+		for(var/i = 0, i<25,i++)
+			target.adjustBruteLoss(-4)
+			target.adjustOxyLoss(-4)
+			target.adjustFireLoss(-4)
+			sleep(10)
+	feedback_add_details("changeling_powers","SRS")
```
Comment (FlavoredCactus) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: Where was this copy pasted from? "SRS" should be diffrent from every single feedback_add_details proc.

**Comment by Ahbahl** [comment]
Didnt copy paste it. RS was for resonant shriek. HS was for hallucination sting. So I took SRS for slow regen sting. Let me browse the code to see if something else has it.

**Comment by Ahbahl** [comment]
Hue, one admin verb has RR which is also for fleshmend so I that can overlap I think. But I checked all feedback_add_details() and none uses SRS. And I mean all. Adminverbs, objective stuff, slime core stuff, everything.

**Comment by FlavoredCactus** [comment]
:+1:

**Pull request merged**